frontend/src/components/uam/helpers/get-columns-helper/get-columns.helper.ts

```diff
@@ -1,25 +1,27 @@
 import { UsersGetAllItemResponseDto } from 'common/types/types';
+import {
+  UserTableAccessor,
+  UserTableHeader,
+} from 'components/uam/common/enums/enums';
 import { Column } from 'react-table';
 
-import { UserTableHeader } from '../../common/enums/enums';
-
 const getColumns = (): Column<UsersGetAllItemResponseDto>[] => {
   return [
     {
       Header: UserTableHeader.ID,
-      accessor: UserTableHeader.ID,
+      accessor: UserTableAccessor.ID,
     },
     {
       Header: UserTableHeader.FULL_NAME,
```

Comment:
```suggestion
      Header: 'Name',
```
Header should be a string

Reply:
Agreed, this enum can't be reused